src/main/java/spireMapOverhaul/zones/heavenlyClouds/HeavenlyCloudsZone.java

```diff
@@ -36,17 +43,39 @@ public Color getColor() {
         return Color.SKY.cpy();
     }
 
+    @Override
+    public boolean canSpawn() {
+        // Don't spawn in Act 3
+        return AbstractDungeon.actNum < 3;
```

Comment:
Hmm, maybe work with modulo so it doesn't stop spawning for endless?

Reply:
There are convenient helper functions for this.
```suggestion
        return this.isAct(1) || this.isAct(2);
```
(I wrote the suggestion to check act 1 or 2 just in case anything weird happens with act numbers.)

Reply:
changed to use this check